### DISCUSSION TITLE
Web Push Notifications

### DISCUSSION THREAD

**Discussion by maevius669** [modrinth/.github#116] [Feature requests]
Getting web push notifications about our followed projects.

**Comment by triphora** [comment]
This is actually kind of impossible I'm afraid. Some folks looked into it and the cost (both in spent effort and in actual dollars) to do this is way too high for us to even consider.

**Reply by BasiqueEvangelist** [reply]
What do you mean? Web Push should be rather simple to implement, there are even crates available that do signing and encryption for you

**Reply by ryanccn** [reply]
The [Web Push Protocol](https://web.dev/push-notifications-web-push-protocol/) is an open protocol that any server can implement by itself, and there are a lot of client/server libraries out there
